requirements.txt

```diff
@@ -10,3 +10,6 @@ zarr
 fsspec
 s3fs
 xarray-datatree==0.0.6
+psutil==5.9.1
+pandas==1.4.3
+more_itertools==8.13.0
```

Comment:
What is `more_itertools`? Where did you use that?

Reply:
[`more_itertools`](https://github.com/more-itertools/more-itertools) is a stable package that uses Python's `itertools` to produce recipes for Python iterables. I personally use it within `Parsed2Zarr` to evenly chunk unique times so that the smallest and largest chunk differ by at most 1 element. This ensures that we will have load balanced chunks.

Reply:
Ah I see. I missed that. 😬  haha thanks for explaining. That's really cool.

Reply:
No worries. I agree, it is cool! I was pleasantly surprised when I found a python library that has distribution algorithms.